public/app/plugins/datasource/grafana-azure-monitor-datasource/partials/query.editor.html

```diff
@@ -57,12 +57,12 @@
       <div class="gf-form gf-form--grow aggregation-dropdown-wrapper">
         <label class="gf-form-label query-keyword width-9">Aggregation</label>
         <div class="gf-form-select-wrapper gf-form-select-wrapper--caret-indent">
-          <select class="gf-form-input width-11" ng-model="ctrl.target.azureMonitor.aggregation" ng-options="f as f for f in ctrl.target.azureMonitor.aggOptions"
+          <select class="gf-form-input width-11" ng-model="ctrl.target.azureMonitor.aggregation" ng-options="f as f for f in ctrl.target.aggOptions"
```

Comment:
:-( That's my mistake. I got hit with a huge merge conflict in this file that got reverted... it took me a long time to recover from that, and apparently I still missed a few. Since this file has no compile time type safety, I appreciate your detailed review of it.